include/sensors/core/message_handlers/sensors.hpp

```diff
@@ -57,7 +57,15 @@ class SensorHandler {
     }
 
     void visit(const can::messages::TipStatusQueryRequest &m) {
-        client.send_tip_notification_queue(m);
+        switch (m.sensor_id) {
+            case can::ids::SensorId::S0: {
+                client.send_tip_notification_queue_rear(m);
+                break;
+            }
+            default:
```

Comment:
definitely use an explicit `case` here - the default can send an error or just ignore the message